cpp/include/libxtreemfs/file_handle_implementation.h

```diff
@@ -49,6 +53,86 @@ class UUIDIterator;
 class UUIDResolver;
 class Volume;
 class XCapManager;
+class VoucherManager;
```

Comment:
The term "voucher" seems - as already discussed - not reflect the actual implementation anymore. We may discuss about a new name. Hope renaming does not generate too much effort.